homeassistant/components/cast/media_player.py

```diff
@@ -150,6 +150,10 @@ def _async_create_cast_device(hass: HomeAssistantType, info: ChromecastInfo):
     hass: HomeAssistantType, config: ConfigType, async_add_entities, discovery_info
 ):
     """Set up the cast platform."""
+    _LOGGER.info(
+        f"pychromcast version: {pychromecast.__version__}, zeroconf version: {pychromecast.discovery.zeroconf.__version__}"
+    )
```

Comment:
We shouldn't info print such info.

It is also not necessary. Home Assistant pins versions, so we know what pychromecast version we have installed.

Reply:
OK, maybe this was stupid.
Does Home Assistant print its own version though?

Reply:
In UI you can go to dev tools -> info.